github/__init__.py

```diff
@@ -227,12 +228,23 @@ async def repositories(self) -> list[Repository]:
                 except KeyError:
                     return result
 
-    async def get_check_suites(self, ref: str) -> list:
-        # https://docs.github.com/en/rest/reference/checks#list-check-suites-for-a-git-reference
-        response = await self.get(f"/repos/{self.repo_path}/commits/{ref}/check-suites")
-        GithubError.raise_from_message(response)
+    async def get_check_runs(self, ref: str) -> list:
+        """
+        Returns all check runs for the commit ref
+        """
 
-        if "check_suites" not in response:
-            raise KeyError
+        async def _get_check_runs(per_page: int, page: int) -> tuple[list, int]:
+            # https://docs.github.com/en/rest/reference/checks#list-check-runs-for-a-git-reference
+            response = await self.get(
+                f"/repos/{self.repo_path}/commits/{ref}/check-runs?per_page={per_page}&page={page}",
+            )
+            GithubError.raise_from_message(response)
+            return response["check_runs"], response["total_count"]
+
+        check_runs, total_count = await _get_check_runs(per_page=100, page=1)
```

Comment:
Can you extract `100` into a variable? This makes it clear that all places where `100` appears actually have to match.

Reply:
I did it in the first version of code but then decided not to create a new variable in a small part of the code that will never be updated afterwards. Added.